test/built-ins/Symbol/species/Symbol.species.notChangedByExtends.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ Symbol.species is retained on subclassing
+es6id: 
+author: Sam Mikes
+description: Symbol.species is retained on subclassing
+includes: 
+  - propertyHelper.js
+---*/
+
+class MyRegExp extends RegExp {
+};
+
+assert.sameValue(MyRegExp[Symbol.species], MyRegExp);
```

Comment:
I think this is just normal prototype lookup? But I suppose this is a good happy path test case regardless.